### ISSUE TITLE
Is there a reason for voltage/1000 while power and energy is not /1000?

### ISSUE THREAD

**Issue by AlexKay88** [hthiery/python-fritzhome#63]
In [fritzhomedevicepowermeter.py](https://github.com/hthiery/python-fritzhome/blob/ed3831e27cdb38aca4a4215daa512c0dcb333d01/pyfritzhome/devicetypes/fritzhomedevicepowermeter.py) you define
`self.power = int(val.findtext("power"))` (---> gives value 25123 for real 25.123 W)
`self.energy = int(val.findtext("energy"))` (---> gives value 999123 for real 999.123 kWh)
but voltage is /1000 (---> gives value 230.123 for real 230.123 V)
https://github.com/hthiery/python-fritzhome/blob/ed3831e27cdb38aca4a4215daa512c0dcb333d01/pyfritzhome/devicetypes/fritzhomedevicepowermeter.py#L36

Is there a reason for that? I would expect real float values (230.123 V) for all or int values (230123 V) for all, but not mixed.

**Comment by hthiery** [comment]
From the AVM Homeautomation (AHA) Documentation:
```
<powermeter>
 <power>Wert in 0,001 W (aktuelle Leistung, wird etwa alle 2 Minuten aktualisiert)
 <energy>Wert in 1.0 Wh (absoluter Verbrauch seit Inbetriebnahme)
 <voltage>Wert in 0,001 V (aktuelle Spannung, wird etwa alle 2 Minuten aktualisiert)
```

Thus it seems to be a bug. I would prefer to have all the values in float and divieded by 1000. Feel free to create a PR.

**Comment by AlexKay88** [comment]
Thanks for your quick reply!
I have no overview and understanding of the whole code, but it may simply be changing 2 lines:
https://github.com/hthiery/python-fritzhome/blob/ed3831e27cdb38aca4a4215daa512c0dcb333d01/pyfritzhome/devicetypes/fritzhomedevicepowermeter.py#L33
to `self.power = float(int(val.findtext("power")) / 1000)`

https://github.com/hthiery/python-fritzhome/blob/ed3831e27cdb38aca4a4215daa512c0dcb333d01/pyfritzhome/devicetypes/fritzhomedevicepowermeter.py#L34
to `self.energy = float(int(val.findtext("energy")) / 1000)`

**Comment by AlexKay88** [comment]
@mib1185 meanhwile created PR https://github.com/hthiery/python-fritzhome/pull/64 for a change the other way around:
Remove the division by 1000 on voltage value, since all other powermeter values are also returned "native"

This will change
`self.voltage = float(int(val.findtext("voltage")) / 1000)`
to
`self.voltage = int(val.findtext("voltage"))`

According to AVM documentation
 <power>Wert in 0,001 W (aktuelle Leistung, wird etwa alle 2 Minuten aktualisiert)
 <energy>Wert in 1.0 Wh (absoluter Verbrauch seit Inbetriebnahme)
 <voltage>Wert in 0,001 V (aktuelle Spannung, wird etwa alle 2 Minuten aktualisiert)
I follow @mib1185 argument that all values should be native as documented by AVM.

**Comment by flabbamann** [comment]
This is fixed by #64 and can be closed 😊
